co2-sensor/main.py

```diff
@@ -0,0 +1,33 @@
+import subprocess
+import re
+import time
+
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+
+PUSHGATEWAY_ADDRESS = "localhost:9091"
+
+
+def main():
+    for i in range(0, 100, 1):
+        co2_conce = get_co2_conce()
+        send_co2_conce(co2_conce)
+        print(i, "分経過: ", co2_conce)
+        time.sleep(60)
+
+
+def get_co2_conce():
```

Comment:
nits: typingを使ってtype hintを書く場合は、return valueにも書くことができます

```suggestion
def get_co2_conce() -> int:
```

Reply:
対応しました。